pkg/api/handlers/libpod/manifests.go

```diff
@@ -333,13 +333,14 @@ func ManifestPush(w http.ResponseWriter, r *http.Request) {
 	decoder := r.Context().Value(api.DecoderKey).(*schema.Decoder)
 
 	query := struct {
-		All               bool   `schema:"all"`
-		CompressionFormat string `schema:"compressionFormat"`
-		CompressionLevel  *int   `schema:"compressionLevel"`
-		Format            string `schema:"format"`
-		RemoveSignatures  bool   `schema:"removeSignatures"`
-		TLSVerify         bool   `schema:"tlsVerify"`
-		Quiet             bool   `schema:"quiet"`
+		All               bool     `schema:"all"`
+		CompressionFormat string   `schema:"compressionFormat"`
+		CompressionLevel  *int     `schema:"compressionLevel"`
+		Format            string   `schema:"format"`
+		RemoveSignatures  bool     `schema:"removeSignatures"`
+		TLSVerify         bool     `schema:"tlsVerify"`
+		Quiet             bool     `schema:"quiet"`
+		AddCompression    []string `schema:"addCompression"`
```

Comment:
this would need an update to the swagger doc to list the new option?

Reply:
done.